pkg/sink/kafka/sarama.go

```diff
@@ -29,8 +29,9 @@ import (
 )
 
 var (
-	defaultKafkaVersion = sarama.V2_0_0_0
-	maxKafkaVersion     = sarama.V2_8_0_0
+	defaultKafkaVersion   = sarama.V2_0_0_0
+	maxKafkaVersion       = sarama.V2_8_0_0
+	keepConnAliveInterval = 5 * time.Minute
```

Comment:
It is better to use it locally.

Reply:
Yes, but this is the better way to share a interval to all AdminClient, SyncProducer and AsyncProducer.